gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemHTTPClientNewIntegrationTest.java

```diff
@@ -0,0 +1,68 @@
+package com.google.cloud.hadoop.gcsio;
+
+import static com.google.cloud.hadoop.gcsio.integration.GoogleCloudStorageTestHelper.getStandardOptionBuilder;
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.auth.Credentials;
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorageFileSystemOptions.ClientType;
+import com.google.cloud.hadoop.gcsio.integration.GoogleCloudStorageTestHelper;
+import com.google.cloud.hadoop.util.RetryHttpInitializer;
+import java.io.IOException;
+import java.net.URI;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Test suites of integration tests for {@link GoogleCloudStorageFileSystemImpl} class configured to
+ * use Apiary client to connect to gcs server.
+ */
+@RunWith(JUnit4.class)
+public class GoogleCloudStorageFileSystemHTTPClientNewIntegrationTest
```

Comment:
nit: do not capitalize `Http`: `GoogleCloudStorageFileSystemHttpClientNewIntegrationTest`

Reply:
Done.